package.json

```diff
@@ -24,23 +24,23 @@
   "repository": "simonhaenisch/prettier-plugin-organize-imports",
   "homepage": "https://github.com/simonhaenisch/prettier-plugin-organize-imports#readme",
   "peerDependencies": {
-    "@volar/vue-language-plugin-pug": "^1.0.4",
-    "@volar/vue-typescript": "^1.0.4",
+    "@vue/language-core": "^2.0.13",
+    "@vue/language-plugin-pug": "^2.0.13",
     "prettier": ">=2.0",
     "typescript": ">=2.9"
   },
   "peerDependenciesMeta": {
-    "@volar/vue-language-plugin-pug": {
+    "@vue/language-core": {
       "optional": true
     },
-    "@volar/vue-typescript": {
+    "@vue/language-plugin-pug": {
       "optional": true
     }
   },
   "devDependencies": {
     "@types/node": "20.8.10",
-    "@volar/vue-language-plugin-pug": "1.0.9",
```

Comment:
Where did this go in the dev dependencies? didn't we have a test case that was using this plugin?

Reply:
If pug is not working without it, it should be changed to https://www.npmjs.com/package/@vue/language-plugin-pug cause `@volar/vue-language-plugin-pug` is deprecated
It might be needed somehow so imports get not lost when using `<template lang="pug">`